load.php

```diff
@@ -236,7 +240,13 @@ function perflab_can_load_module( $module ) {
 	}
 
 	// Call the closure to determine whether the module can be loaded.
-	return (bool) $module();
+	$result = $module();
+
+	if ( false !== $result && true !== $result ) {
+		return $get_message ? $result : false;
+	}
+
+	return (bool) $result;
```

Comment:
See above, this should be reverted, as it makes the API unnecessarily messy.

Reply:
Done